### PR TITLE
Alkimi Bid Adapter : add handling for user-sync URL and pass custom object with bid request

### PR DESCRIPTION
**Pull request by kalidas-alkimi**
## Type of change

- [ ] Feature

- [No] Does this change affect user-facing APIs or examples documented on http://prebid.org?

### CODE REVIEW

**Comment by kalidas-alkimi** [comment]
Hi 

I am not sure why the e2etest is failing,  Could you please check this and help on this failure? if it's possible could you please rerun the e2etest from your side? thanks.

**Comment by ChrisHuie** [comment]
> Hi
> 
> I am not sure why the e2etest is failing, Could you please check this and help on this failure? if it's possible could you please rerun the e2etest from your side? thanks.

Sometimes the e2e testing is flaky. Don't worry about it.

**Comment by kalidas-alkimi** [comment]
Hello

We are waiting for his change to be merged, Could you please check and approve it? Thanks.

**Review by robertrmartinez — CHANGES_REQUESTED** [review]

File: modules/alkimiBidAdapter.js [hunk 1]
```diff
@@ -141,6 +152,24 @@ export const spec = {
...
     return false;
+  },
+
+  getUserSyncs: function(syncOptions, serverResponses, gdprConsent) {
+    if (syncOptions.iframeEnabled && serverResponses.length > 0) {
+      const serverBody = serverResponses[0].body;
+      if (!serverBody || typeof serverBody !== 'object') return [];
+
+      const { iframeList } = serverBody;
+      if (!iframeList || typeof iframeList !== 'object') return [];
```
[the remark this answers is not in the document]
Comment (kalidas-alkimi) [reply modules/alkimiBidAdapter.js]: updated

[the remark this answers is not in the document]
Comment (kalidas-alkimi) [reply modules/alkimiBidAdapter.js]: @robertrmartinez   Could you please check the latest changes and approve? Thanks.

**Comment by kalidas-alkimi** [comment]
Hello  @robertrmartinez  / @ChrisHuie 

Could you please check this change and approve it? We are completely blocked deliver this change. Thanks.

**Review by robertrmartinez — APPROVED** [review]

**Pull request merged**